<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BoutonCreerContact">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Créer Contact</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>1271</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #FFFFFF;
alternate-background-color: #BEEAFD;
selection-background-color:#1F567C;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prenom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Photo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telephone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date de création</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BoutonSupprimerContact">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer Contact</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CritereRecherche">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rechercher par</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entreprise</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Telephone</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="BoutonRechercher">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ValeurRecherche">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 800 24pt &quot;Cantarell&quot;;</string>
    </property>
    <property name="text">
     <string>LISTE DE CONTACTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonRechercheDates">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher entre deux dates</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateFin">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>390</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateDebut">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>390</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BoutonNombreContact">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de contact</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEXPORT">
    <property name="title">
     <string>EXPORT</string>
    </property>
    <addaction name="actionExport_JSON"/>
   </widget>
   <addaction name="menuEXPORT"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_JSON">
   <property name="text">
    <string>Export JSON</string>
   </property>
  </action>
  <action name="actionNombre_de_contacts">
   <property name="text">
    <string>Nombre de contacts</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionInteraction">
   <property name="text">
    <string>Interaction</string>
   </property>
  </action>
  <action name="actionRechercher_entre_deux_dates">
   <property name="text">
    <string>Rechercher entre deux dates</string>
   </property>
  </action>
  <action name="actionNombre_de_contact">
   <property name="text">
    <string>Nombre de contact</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
